definitions/npm/statuses_v1.5.x/flow_v0.82.x-/statuses_v1.5.x.js

```diff
@@ -0,0 +1,9 @@
+declare module "statuses" {
+  declare module.exports: {
+    (string | number): number,
+    [string | number]: ?(string | number),
+    redirect: { [number]: ?boolean },
+    empty: { [number]: ?boolean },
+    retry: { [number]: ?boolean }
```

Comment:
If you would want to go fancy, this could perhaps have fixed set of all the standard status codes.